src/Place/ReadModel/Duplicate/CleanPlaceName.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\ReadModel\Duplicate;
+
+use CultuurNet\UDB3\Json;
+use CultuurNet\UDB3\Model\ValueObject\Geography\Address;
+
+class CleanPlaceName
+{
+    private const MAX_LENGTH_TITLE = 150;
+    private const NONE = 'none';
+
+    public static function transform(Address $address, string $title): string
+    {
+        if (mb_strtolower($title) === self::NONE) {
+            return '';
+        }
+
+        if (mb_strlen($title) > self::MAX_LENGTH_TITLE) {
+            return '';
+        }
+
+        if (str_contains($title, $address->getStreet()->toString())) {
+            return '';
+        }
+
+        $title = htmlspecialchars($title);
+
+        $title = str_replace([' BE', 'BE ', ' NL', 'NL '], [' Belgium', 'Belgium ', ' Netherlands', 'Netherlands '], $title);
```

Comment:
I think this replacement is done on the address part and not on the location name.

Reply:
it is not, it is done on location name to prevent double information in the google api call, given that we just concat location name and address.
I did make the code work for more countries.